lib/android.js

```diff
@@ -29,13 +29,23 @@ function getApi () {
       },
       '_ZN3art6Thread14CurrentFromGdbEv': ['art::Thread::CurrentFromGdb', 'pointer', []],
       '_ZNK3art6Thread13DecodeJObjectEP8_jobject': ['art::Thread::DecodeJObject', 'pointer', ['pointer', 'pointer']],
-      '_ZN3art6mirror6Object5CloneEPNS_6ThreadE': ['art::mirror::Object::Clone', 'pointer', ['pointer', 'pointer']]
+      '_ZN3art6mirror6Object5CloneEPNS_6ThreadE' : function(address) {
+        this["art::mirror::Object::Clone"] = new NativeFunction(address, 'pointer', ['pointer', 'pointer']);
+      },
+      '_ZN3art6mirror6Object5CloneEPNS_6ThreadEm' : function(address) {
+        const nativeFn = new NativeFunction(address, 'pointer', ['pointer', 'pointer', 'int']);
+        this["art::mirror::Object::Clone"] = function(thisptr, threadptr) {
+          return nativeFn(thisptr, threadptr, 0);
+        }
+      }
     },
     variables: {
       '_ZN3art7Runtime9instance_E': function (address) {
         this.runtime_instance_ptr = address;
       }
-    }
+    },
+    optionals : ['_ZN3art6mirror6Object5CloneEPNS_6ThreadEm',
+      '_ZN3art6mirror6Object5CloneEPNS_6ThreadE']
```

Comment:
Since we're changing the data structure let's use a `Set` here instead:

```js
    optionals: new Set([
      '_ZN3art6mirror6Object5CloneEPNS_6ThreadEm',
      '_ZN3art6mirror6Object5CloneEPNS_6ThreadE',
    ]);
```